WebHostLib/static/styles/tracker__HK.css

```diff
@@ -0,0 +1,260 @@
+@import url('https://fonts.googleapis.com/css2?family=Lexend+Deca:wght@100..900&display=swap');
+
+.tracker-container {
+    width: 600px;
+    box-sizing: border-box;
+    font-family: "Lexend Deca", Arial, Helvetica, sans-serif;
+    border: 2px solid black;
+    border-radius: 4px;
+    resize: both;
+
+    background-color: #000000;
+    color: white;
+}
+
+/* .tracker-container.charms {
+    width: 600px;
+} */
+
+/* .tracker-container ::before {
+    background-image: url('https://cdn.wikimg.net/en/hkwiki/images/9/96/Dialogue_Top.png');
+    background-position: center;
+    background-size: contain;
+    background-repeat: no-repeat;
+    top: 0;
+}
+
+.tracker-container ::before {
+    position: absolute;
+    content: "";
+    height: 2rem;
+    width: calc(100% - 2rem);
+} */
+
+.hidden {
+    display: none !important;
+}
+
+h2 {
+    text-transform: none;
+    cursor: unset;
+    color: #fff677;
+    margin-bottom: 2px;
+    margin-top: 33px;
+    font-size: 44px;
+}
+
+h3 {
+    color: #ffef00;
+    font-weight: bold;
+}
+
+.notRandomized {
+    background-color: #d0d0d0;
+}
+
+.elevatorPass, .Rightslash {
+    transform: rotate(-90deg);
+}
+
+.Leftslash {
+    transform: rotate(90deg);
+}
+
+.Downslash {
+    transform: rotate(0deg);
+}
+
+.Upslash {
+    transform: rotate(180deg);
+}
+
+.fullNail {
+    transform: rotate(225deg);
+}
+
+/* html::before {
+    background-image: url('https://cdn.wikimg.net/en/hkwiki/images/thumb/2/2f/VHpromo6.png/624px-VHpromo6.png?20190823174122');
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: center;
+} */
```

Comment:
i assume this was commented while you were developing and can just be deleted now?

Reply:
Oh yes. I'll remove that.